dotnet/Workloads/vs-workload.template.props

```diff
@@ -1,14 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
   <PropertyGroup>
-    <TargetName>Microsoft.NET.Sdk.MaciOS.Workload</TargetName>
+    <TargetName>Microsoft.NET.Sdk.MacCatalyst.macOS.Workload</TargetName>
     <ManifestBuildVersion>@IOS_WORKLOAD_VERSION@</ManifestBuildVersion>
   </PropertyGroup>
   <ItemGroup>
     <!-- Shorten package names to avoid long path caching issues in Visual Studio -->
-    <ShortNames Include="@PACK_VERSION_LONG@">
-      <Replacement>@PACK_VERSION_SHORT@</Replacement>
-    </ShortNames>
```

Comment:
We should keep this replacement, as it should be used for `MacCatalyst` packages.

Reply:
Fixed